src/Crypto/RNCryptor/V3/Decrypt.hs

```diff
@@ -94,6 +92,13 @@ removePaddingSymbols input =
   in B.take (B.length input - fromEnum lastWord) input
 
 --------------------------------------------------------------------------------
+decrypt_ :: AES128 -> ByteString -> ByteString -> ByteString
+decrypt_ a iv cipherText =
+  cbcDecrypt a iv' cipherText
+  where
+    iv' = maybe (error "decrypt_ failed makeIV") id $ makeIV iv
```

Comment:
you can use `fromMaybe` instead of `maybe blabla id`, they are equivalent. Usually tools like `hlint` suggest this automatically to you and can be used with most popular editors. If you are not using it, very recommended! 😉